yafti/__init__.py

```diff
@@ -1,5 +1,5 @@
 """
-Copyright 2023 Marco Ceppi
+Copyright 2024 Alex Banna
```

Comment:
You can keep multiple copyright headers. My copyright doesn't go away because theyre's a new maintainer. Add you name below and the year it started.

Reply:
sorry